spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JGitEnvironmentRepository.java

```diff
@@ -484,6 +486,14 @@ private boolean containsBranch(Git git, String label, ListMode listMode)
 		return false;
 	}
 
+	protected void warn(String message, Exception ex) {
+		if(logger.isDebugEnabled()) {
+			logger.warn(message, ex);
```

Comment:
why? 
if debug enabled new data will be sent but message level will stay as warn. If I search for errors and warnings in kibana I will not find this message anymore, it will be confusing.